docs/kopia_troubleshooting.md

```diff
@@ -123,4 +114,184 @@ Users may also find the following type of errors in the logs while maintenance i
   * `Error getting a backup store`
   * `BackupStorageLocation is invalid`
 
-     
\ No newline at end of file
+## Kopia Repository Maintenance
+
+* repository maintenance commands:
+```
+kopia maintenance info
+kopia maintenance run 
+kopia maintenance run --full
```

Comment:
It would be also nice to show the snapshot command and ask to compare with the number of velero backups. There should be N-N number mapping between velero backups for a given namespace with the repository used:

```shell
kopia snapshot ls --all --storage-stats
```

Additionally it would be interesting to see the content stats before and after maintenance tasks:

```shell
kopia content stats
```

Reply:
I have that in the doc already :) line 105